README.md

```diff
@@ -1,94 +1,187 @@
-# The ops library
+# The `ops` library
 
-<!-- The text below is also at the top of ops/__init__.py. Keep in sync! -->
 
-The ops library is a Python framework ([`available on PyPI`](https://pypi.org/project/ops/)) for developing
-and testing [Juju](https://juju.is/) charms in a consistent way, using standard Python constructs
-to allow for clean, maintainable, and reusable code.
+The `ops` library is a Python framework for developing and testing charms in a consistent way, using standard Python constructs to allow for clean, maintainable, and reusable code. It is an official component of the Charm SDK, itself a crucial part of the Juju universe.
 
-A charm is an operator -- business logic encapsulated in a reusable software
-package that automates every aspect of an application's life.
+> ✔️ `ops` is  ([`available on PyPI`](https://pypi.org/project/ops/)).
+>
+> ⚠️ The latest version of `ops` requires Python 3.8 or above.
+> 
+> 🥇 While charms can be written in any language, `ops` defines the latest standard. All new charms are encouraged to use it.
 
-Charms written with ops support Kubernetes using Juju's "sidecar charm"
-pattern, as well as charms that deploy to Linux-based machines and containers.
 
-Charms should do one thing and do it well. Each charm drives a single
-application and can be integrated with other charms to deliver a complex
-system. A charm handles creating the application in addition to scaling,
-configuration, optimisation, networking, service mesh, observability, and other
-day-2 operations specific to the application.
 
-The ops library is part of the Charm SDK (the other part being Charmcraft).
-Full developer documentation for the Charm SDK is available at
-https://juju.is/docs/sdk.
+||||
+|-|-|- |
+|| Learn about Juju! | Start deploying charms: <br> https://juju.is/docs/juju <br> _Juju is declarative, model-driven, and has native support for integrations and scaling, on any cloud. You want to deploy `foo`? Run `juju deploy foo`! You wish to integrate `foo` and `bar`? Run `juju integrate foo bar`!_ |
+||||
+|| Get better at Juju! | Take advantage of the charm ecosystem: <br> https://charmhub.io/ <br> _Quickly deploy a cluster ([OpenStack](https://charmhub.io/openstack-base), [Kubernetes](https://charmhub.io/charmed-kubernetes)), a database ([PostgreSQL](https://charmhub.io/postgresql-k8s), [MongoDB](https://charmhub.io/mongodb), etc.), an observability stack ([Canonical Observability Stack](https://charmhub.io/cos-lite)), an MLOps solution ([Kubeflow](https://charmhub.io/kubeflow)), and so much more!_ |
+||||
+|:point_right:| **Make Juju your own!** | Write your own charm: <br> https://juju.is/docs/sdk <br> _Juju is written in Go, but our SDK supports easy charm development in Python!_ |
```

Comment:
I think this is a bit unintuitive:

- The first row starts with "Learn about Juju!" but then says "Start deploying charms", then links to learn-about-Juju docs, then describes Juju briefly. 3/4 of these are learning about Juju -- "Start deploying charms" doesn't seem to fit.
- I also wonder if we could link the text rather than have separate links. "Learn about Juju!" could be the link to the docs. Other approaches are possible too.
- Second row starts with "Get better at Juju", but then links to Charmhub and individual charms. Maybe "Install charms from Charmhub!" (linked)
- Similar with the third row -- could we reduce the number of "things" by linking "Learn how to write your own charm" or similar?
- I like the enthusiasm, but I think there are a few too many breathless exclamation points here. Maybe remove those in the left column.

Reply:
I'll drop the exclamation marks.
Happy to link the text instead of having separate links.
Regarding the titles -- I was trying to create a simple narrative around the Juju-world entrypoint, Juju, while also clarifying the connection to charms. Hence the Juju-based titles ("Get started with Juju", "Get better at Juju", "Make Juju your own") and the charm-based subtitles ("Start deploying charms", "Take advantage of the charm ecosystem", and "Write your own charm"). I'm still tempted by that by agree the way I've implemented right now feels forced.